packs/_source/classfeatures/ranger/ranger-features/vanish.json

```diff
@@ -98,7 +98,7 @@
     "prerequisites": {
       "level": null
     },
-    "identifier": "vanish"
+    "identifier": "natures-veil"
```

Comment:
What criteria are we using to determine if a feature has been renamed vs. replaced?

Reply:
D&D Beyond's updated terminology list :)

https://www.dndbeyond.com/posts/1810-updates-in-the-players-handbook-2024#New_Terminology

Reply:
OK nice, thank you